lib/middleware.js

```diff
@@ -135,6 +135,9 @@ exports.checkQuota = function () {
     if (utils.isServiceFabrikOperation(req.body)) {
       logger.debug('[Quota]: ServiceFabrikOperation : calling next handler..');
       next();
+    } else if (CONST.HTTP_METHOD.PATCH === req.method && utils.isNotPlanUpdate(req.body)) {
```

Comment:
shouldn't this be `req.method === CONST.HTTP_METHOD.PATCH` ?

Reply:
It can be that way as well .

But its a best practice to use constant on the LHS - https://en.wikipedia.org/wiki/Yoda_conditions to avoid programming errors of using assignment operators`.

Reply:
ok.. this is fine then.

Reply:
This check could have gone as an OR clause in Line135 as the code fragments of both the blocks are the same?